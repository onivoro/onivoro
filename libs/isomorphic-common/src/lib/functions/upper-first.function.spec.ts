import { upperFirst } from './upper-first.function';

describe('upperFirst', () => {
  it.each([
    'asdf',
    'XCVB'
  ])('GIVEN %j, does what u expect', (input) => {
    expect(upperFirst(input)).toMatchSnapshot();
  });

  it.each([
    null, undefined, 0, false
  ])('GIVEN falsey input %j, does what u expect', (input) => {
    expect(upperFirst(input)).toMatchSnapshot();
  });
});