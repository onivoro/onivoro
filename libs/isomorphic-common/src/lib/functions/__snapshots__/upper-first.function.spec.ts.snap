// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`upperFirst GIVEN "XCVB", does what u expect 1`] = `"Xcvb"`;

exports[`upperFirst GIVEN "asdf", does what u expect 1`] = `"Asdf"`;

exports[`upperFirst GIVEN falsey input 0, does what u expect 1`] = `0`;

exports[`upperFirst GIVEN falsey input false, does what u expect 1`] = `false`;

exports[`upperFirst GIVEN falsey input null, does what u expect 1`] = `null`;

exports[`upperFirst GIVEN falsey input undefined, does what u expect 1`] = `undefined`;
